tests/unit/multiclass/KNN_unittest.cc

```diff
@@ -226,3 +226,79 @@ TEST(KNN, classify_multiple_kdtree)
 
 
 }
+
+
+TEST(KNN, nearest_neighbours_brute)
+{
+	//simple non-trivial test
+	//initialize labels
```

Comment:
minor nitpick: remove the comment (redundant)

Reply:
Done.